language/stdlib/modules/LibraVersion.move

```diff
@@ -41,19 +50,35 @@ module LibraVersion {
         );
 
         LibraConfig::set<LibraVersion>(
-            account,
+            lr_account,
             LibraVersion { major }
         );
     }
+    spec fun set {
+        include LibraTimestamp::AbortsIfNotOperating;
+        include Roles::AbortsIfNotLibraRoot{account: lr_account}; // Aborts if not LibraRoot [B19].
+        aborts_if LibraConfig::get<LibraVersion>().major >= major with Errors::INVALID_ARGUMENT;
+        include LibraConfig::SetAbortsIf<LibraVersion>{account: lr_account};
+        include LibraConfig::SetEnsures<LibraVersion>{payload: LibraVersion { major }};
+    }
 
     spec module {
         /// After genesis, version is published.
         invariant [global] LibraTimestamp::is_operating() ==> LibraConfig::spec_is_published<LibraVersion>();
 
-        /// The permission "UpdateLibraProtocolVersion" is granted to LibraRoot [B20].
-        invariant [global, isolated] forall addr: address where exists<LibraVersion>(addr):
+        /// The permission "UpdateLibraProtocolVersion" is granted to LibraRoot [B19].
+        invariant [global, isolated] forall addr: address where exists<LibraConfig<LibraVersion>>(addr): //  TODO: Is this a valid invariant although it is related to another module (e.g., LibraConfig).
```

Comment:
Good question. 

It is valid provided we follow the discipline that only this module will update `LibraConfig<LibraVersion>`.

It is interesting to see that normally we get this by construction (only a given module can update the resources it owns), but clearly, LibraConfig kind of circumvents this discipline as it is a general storage module.

Still, the way how we use LibraConfig in the framework, things should be fine, because no one else than `LibraVersion` updates `LibraConfig<LibraVersion>`. However, this is not enforced and only by convention.

PS. General background is that a global invariant effecting the state of a another module is not evaluated unless both the enforcing and state declaring module are verified together. In this case, LibraVersion and LibraConfig.

@shazqadeer @DavidLDill @emmazzz for general global invariant discussion, @tzakian  for the recipes we are using in the framework to work around language restrictions.